testing/bloaty/APKBUILD

```diff
@@ -0,0 +1,42 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+_commitid=2234386bcee7297dfa1b6d8a5d20f95ea4ed9bb0
+_commitid_libfuzzer=e9387f0b6d411e18ca75f08aecfec1238b8e113f
+pkgname=bloaty
+# Fake a version until upstream does one
+# format is simply 0.0.0.0.${commit_count}
+pkgver=0.0_git20170301
+pkgrel=0
+pkgdesc="A size profiler for binaries"
+url="https://github.com/google/bloaty"
+arch="all"
+license="apache"
+depends=""
+makedepends="gcc git"
+# This software has no releases yet, and has git submodules for 3 things
+# I managed to get everything but libFuzzer from a released tarball.
+# Upstream already has an issue from an arch AUR maintainer to fix this.
+source="https://github.com/google/bloaty/archive/${_commitid}.tar.gz
+	https://github.com/google/re2/archive/2017-01-01.tar.gz
+	https://github.com/google/googletest/archive/release-1.8.0.tar.gz
+	byteswap.patch
+	"
+builddir="$srcdir/${pkgname}-${_commitid}"
+
+build() {
+	rmdir third_party/re2 third_party/googletest || return 1	
+	ln -s ../../re2-2017-01-01 third_party/re2 || return 1
+	ln -s ../../googletest-release-1.8.0 third_party/googletest || return 1
+	git clone https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer third_party/libFuzzer || return 1
```

Comment:
i don't like this, it should be possible to build a package without requiring a functioning internet connection.

if we have to git clone during the build process, then it makes the build process much more complex